src/core/border_router/routing_manager.cpp

```diff
@@ -102,6 +108,9 @@ Error RoutingManager::Init(uint32_t aInfraIfIndex, bool aInfraIfIsRunning)
     GenerateOnLinkPrefix();
 
     error = mInfraIf.HandleStateChanged(mInfraIf.GetIfIndex(), aInfraIfIsRunning);
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
+    RequestInfraIfNat64Prefix();
```

Comment:
Is it too early to request for infra net64 prefix in Init.

I think it's better to do it in SetEnabled(true).

Thoughts

Reply:
Acknowledged